File: Zestawy Zadań/Python/4 - Tablice/Zestaw 2 zadania/4.2.15 -  Zgadywanie liczb.py
import random

U = 0
liczbaprob = 0
proby = [0] * 100

L = random.randint(1,1000)

while True:
    U = int(input("Podaj liczbe: "))
    if U < L:
        print("Za mała")
    elif U > L:
        print("Za duza")
    if U == L:
        print("Brawo!")
        print("Twoje proby: ")
        for i in range(0,liczbaprob):
            print(proby[i], end=" ")
        break
    elif liczbaprob == 10:
        print("Game Over Christopher")
        break
    else:
        proby[liczbaprob] = U
        liczbaprob += 1

File: Zestawy Zadań/C++/4 - Tablice/Zestaw 2 zadania/4.2.15 -  Zgadywanie liczb .cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main()
{
    int L;
    int U;
    int liczbaprob = 0;
    int proby[100];
    srand(time(nullptr));
    L = rand() % 1001 - 1;
    
    do
    {
        cout << "Podaj liczbe: ";
        cin >> U;
        if (U < L)
        {
            cout << "Za mala" << endl;
        }
        else if (U > L)
        {
            cout << "Za duza" << endl;
        }
        if (L == U)
        {
            cout << "Brawo! " << endl;
            cout << "Twoje proby:";
            for (int i = 0; i < liczbaprob; i++)
            {
                cout << proby[i] << " ";
            }
        }
        else if (liczbaprob == 100)
        {
            cout << "Game Over christopher" << endl;
            cout << "Wylosowana liczba: " << L << endl;
            return 0;
        }
        else
        {
            proby[liczbaprob] = U;
            liczbaprob++;
        }

    } while (U != L);
    return 0;
}
